tests/server/test_compilerservice.py

```diff
@@ -1013,3 +1022,34 @@ async def test_compileservice_api(client, environment):
 
     result = await client.get_reports(environment, limit=APILIMIT)
     assert result.code == 200
+
+
+async def test_notification_on_failed_exporting_compile(server, client, environment: str) -> None:
+    compilerservice = server.get_slice(SLICE_COMPILER)
+    env = await data.Environment.get_by_id(uuid.UUID(environment))
+
+    result = await client.list_notifications(env.id)
+    assert result.code == 200
+    assert len(result.result["data"]) == 0
+
+    compile_id, _ = await compilerservice.request_recompile(env, False, True, uuid.uuid4())
+
```

Comment:
Improve readability:

```suggestion
    compile_id, _ = await compilerservice.request_recompile(env, force_update=False, do_export=True, remote_id=uuid.uuid4())

```